Makefile

```diff
@@ -12,8 +12,7 @@ GOFLAGS :=
 .PHONY : coverage
 
 dependencies :
-		go get github.com/alecthomas/gometalinter
-		gometalinter --install
+		curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin v1.21.0
```

Comment:
FWIW, we install with `go install -i github.com/golangci/golangci-lint/cmd/golangci-lint`

Reply:
According to the readme of golangci-lint https://github.com/golangci/golangci-lint#binary, it suggests use curl to install.
It says `Please, do not install golangci-lint by go get:`

```
1. go.mod replacement directive doesn't apply. It means you will be using patched version of golangci-lint.
2. it's much slower than binary installation
3. its stability depends on your Go version (e.g. on this compiler Go <= 1.12 bug).
4. it's not guaranteed to work: e.g. we've encountered a lot of issues with Go modules hashes.
5. it allows installation from master branch which can't be considered stable.
```

I think `go install` is the same with with `go get`, it will pollute the go.mod/go.sum files.